lib/navigation/filtering/accelerometer_kalman.cpp

```diff
@@ -0,0 +1,104 @@
+/*
```

Comment:
assuming this file is a WIP?

Reply:
Yeah this file is either getting fixed or deleted for now, still TBD